File: beginner/javascript/1008.js
var input = require('fs').readFileSync('stdin', 'utf8');

var values = input.split('\n');

var numFunc = parseInt(values.shift());
var horasTrab = parseInt(values.shift());
var valorHora = parseFloat(values.shift());

var valorGanho = horasTrab * valorHora;

console.log("NUMBER = " + numFunc + "\nSALARY = U$ " + valorGanho.toFixed(2));

File: beginner/c/1008.c
#include <stdio.h>

int main(){

    int number;
    int numHora;
    double valorHora;

    scanf("%d", &number);
    scanf("%d", &numHora);
    scanf("%lf", &valorHora);

    double salario = numHora * valorHora;

    printf("NUMBER = %d\nSALARY = U$ %.2lf\n", number, salario);

    return 0;
}
